source/docs/prismic/integration-fields.md

```diff
@@ -0,0 +1,289 @@
+
+## Update your types and data in Prismic
+
+Integration fields created in the previous section should now be available in the Custom types editor. You can drag & drop them to your content model in build mode as any other field.
+
+![Custom type field from Integration field](./assets/integration-fields/configure-integration-field.jpg)
+
+See [Configure Integration Fields (Prismic documentation)](https://prismic.io/docs/core-concepts/integration-fields) for details.
+
+In the writing room, Content managers can now create or update documents to select values from Front-Commerce data for the configured fields:
+
+![Selecting a value for a document field using an Integration field](./assets/integration-fields/document-integration-field-lookup.jpg)
+
+## Expose data in your GraphQL schema
+
+This section supposes that you have already exposed data from the previously modified Prismic custom type in your GraphQL schema. It will guide you throughout the process of exposing an additional Integration field data.
+
+**We've tried to make the process as idiomatic as what you might know from implementing custom Front-Commerce features.**
+
+### Add the field to your schema
+
+First, you have to update your GraphQL schema definition with the new field to expose. Instead of exposing an id or a SKU, we will expose an existing GraphQL type for a `product` field.
+
+```diff
+type MyPrismicContent {
+  uid: ID
+  title: String
+  richtext: DefaultWysiwyg
++  product: Product
+}
+```
+
+### Update Prismic types with the `IntegrationFieldTransformer`
+
+You then have to instruct Front-Commerce Prismic module to transform data from Prismic with information from a [registered integration field](#Register-Integration-fields-in-Front-Commerce).
+
+The `IntegrationFieldTransformer` will transform data from [an Integration Field implementation](#Implementations). We can get the correct Integration field instance using the name used when registering the Integration field, using Prismic loader's `getIntegrationField` method.
+
+Here is how it would look to turn a Prismic field named `fc_product` into a Front-Commerce Product:
+
+```diff
+const {
+  TitleTransformer,
++  IntegrationFieldTransformer,
+} = loaders.Prismic.transformers;
+
+const all = await loaders.Prismic.loadSingle("allfields", {
+  fieldTransformers: {
+    // […]
++    fc_product: new IntegrationFieldTransformer(
++      loaders.Prismic.getIntegrationField("Product")
++    ),
+  },
+});
+```
+
+### Create resolvers for these fields
+
+<blockquote class="wip">
+**WIP:** as of version `0.5` of the Prismic module, resolvers must be defined as described below. We will try in a future version of the module to make it work out-of-the-box, without resolver changes if field names match.
+</blockquote>
+
+To prevent fetching data systematically, fields resolve with either a `null` value (field empty in Prismic) or an object with a `raw` value and a `loadValue()` method to actually fetch the related data.
```

Comment:
```suggestion
To prevent fetching data systematically, fields resolve with either a `null` value (empty field in Prismic) or an object with a `raw` value and a `loadValue()` method to actually fetch the related data.
```